### DISCUSSION TITLE
Using LockPlugin maxDays but still be able to repick a new start date?

### DISCUSSION THREAD

**Discussion by giorgiofnool** [easepick/easepick#42] [Q&A]
Hi,

is it possible to pick a new start date after a selection when maxDays is set?
https://jsfiddle.net/vtgd9b1u/

I would like to set a maxDays but still have to possibility to pick a new date without closing and reopening the calendar.
It is possible to select a new date only if it's in range with the selected start date.
Thanks!

**Comment by wakirin** [comment]
The `repick` option requires the use of the `elementEnd` option.

https://jsfiddle.net/waki/ck3y2rj9/

**Reply by giorgiofnool** [reply]
Sorry, I used the `repick` option without knowing how it works.

But what I'm trying to do is different.
I would like that the user could click on an "out of range" date to set a new starting day.

For example:
With maxDays = 7 if I click on the 1st then I can click up to the 7th because starting from the 8th the dates are grayed out.
What about if the user could click the 10th and set a new starting date from that?

Thanks.

**Reply by wakirin** [reply]
Perhaps I misunderstood your case.
Don't use the `repick` option If you need the user to be able to pick a different start date outside of your limit (7)
Or use `AmpPlugin` and enable the `resetButton` option so that the user can clear selection and select any available start date.

**Reply by giorgiofnool** [reply]
The provided solutions works. I was looking for a more "intuitive" mode but anyway thanks.
